var/spack/repos/builtin/packages/mesa/package.py

```diff
@@ -241,7 +248,7 @@ def meson_args(self):
         if "+egl" in spec:
             num_frontends += 1
             args.extend(["-Degl=enabled", "-Dgbm=enabled", "-Ddri3=enabled"])
-            args_platforms.append("surfaceless")
+            # args_platforms.append("surfaceless")
```

Comment:
What's the motivation for this change? Does it not work, or does it prevent something from working?

Reply:
I believe I couldn't get this argument to work at all and didn't clean up properly. Do you happen to know what the `surfaceless` platform is/requires?